### PR TITLE
TEST-#20: Add linters CI

### PR DESCRIPTION
**Pull request by prutskov**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  lint-black:
+    name: lint (black)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8.x"
```
Comment (YarShev) [inline .github/workflows/ci.yml:32]: Why do we want to use 3.8.x instead of 3.7.x?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  lint-flake8:
+    name: lint (flake8)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8.x"
```
Comment (YarShev) [inline .github/workflows/ci.yml:46]: same question

File: .github/workflows/ci.yml (hunk 1 above)
Comment (prutskov) [reply .github/workflows/ci.yml:32]: 3.8.x has longer period of support than 3.7.x. This frees us from updating the version of python for those checks in the future

Comment (YarShev) [reply .github/workflows/ci.yml:32]: I believe that many people have still been working in env with python 3.7.x and testing their code with it. So I prefer to move to python 3.8.* with whole community.

Comment (prutskov) [reply .github/workflows/ci.yml:32]: Including version 3.8.x in our linter checks we don't limit any way user to use others actual versions of python for developing. I prefer to leave it as it is.

Comment (YarShev) [reply .github/workflows/ci.yml:32]: I checked `black` and `flack8` support py3 so there shouldn't be any issue with minor versions. Let's leave as it is.

**Review by YarShev — COMMENTED** [review]
LGTM

**Pull request merged**